src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -179,7 +179,11 @@ const ApplicationViewWrapper: React.FC<ApplicationViewWrapperProps> = (props) =>
           margin: 5,
         }}
       >
-        {parametersReady && <Form.Field required={isRequired}>{PluginComponent}</Form.Field>}
+        {parametersReady ? (
+          <Form.Field required={isRequired}>{PluginComponent}</Form.Field>
+        ) : (
+          <Loader active inline />
```

Comment:
Thanks! That'a already a bit better than before

Reply:
Yeah, I'd like to implement a more thorough solution to incrementally show evaluated elements, but this'll do for now :)